include/linux/mptcp_upstream.h

```diff
@@ -36,6 +36,7 @@ enum {
 /* netlink interface */
 #define MPTCP_PM_NAME		"mptcp_pm"
 #define MPTCP_PM_CMD_GRP_NAME	"mptcp_pm_cmds"
+#define MPTCP_PM_EV_GRP_NAME    "mptcp_events"
```

Comment:
Wasn't this supposed to be `mptcp_pm_events`?

Reply:
Yeah, Matthieu was going to tweak that

Reply:
Latest export branch tag does have the `mptcp_pm_events` change

Reply:
Reference: https://github.com/multipath-tcp/mptcp_net-next/commit/1c2dd8ae2f250f7289500962883e568d41378606

Reply:
Got it.  Thanks!

Reply:
Thanks to your PR, I just noticed the name was not properly aligned with the others. Just fixed: https://github.com/multipath-tcp/mptcp_net-next/commit/1f2bcf78ff83 (will soon land on the export branch)